frontend/src/scenes/trends/mathsLogic.tsx

```diff
@@ -264,6 +480,25 @@ export const mathsLogic = kea<mathsLogicType>({
                 return allMathOptions
             },
         ],
+        selectFormattedOptions: [
+            (s) => [s.groupsAccessStatus, s.groupsMathFormattedSelectDefinitions],
+            (groupsAccessStatus, groupsMathFormattedSelectDefinitions): LemonSelectSection<string>[] => {
+                const hasGroupAccess = [
+                    GroupsAccessStatus.HasAccess,
+                    GroupsAccessStatus.HasGroupTypes,
+                    GroupsAccessStatus.NoAccess,
+                ].includes(groupsAccessStatus)
+                const mathOptions = SELECT_FORMATTED_OPTIONS
+
+                if (!hasGroupAccess) {
+                    mathOptions[0].footer = <GroupIntroductionFooter />
+                } else {
+                    mathOptions[0].options.push(...groupsMathFormattedSelectDefinitions)
+                }
```

Comment:
As I was making changes here I wasn't sure if the selection to show the notice is correct.

Currently to me it looks like "You have group access if your status is `HasAccess`, `HasGroupTypes` or `NoAccess`"

Something there feels wrong...

Reply:
Yup, I have it backwards

Reply:
I think the intent here is that if the status is any of those 3, show the intro because it means that groups aren't fully used yet, otherwise it's `AlreadyUsing` which you then populate the group options

Reply:
Ah okay gotcha. That wasn't super clear but I guess it makes sense

Reply:
Also I think you had it right but I inverted it when I made some changes 😅